### PR TITLE
Feature/#23 missing timeline and user features

### PR DESCRIPTION
**Pull request by Lucki2g**
#23 & #31 
Unfollow and follow buttons
add message field
<usersname>'s timeline

### CODE REVIEW

File: src/controllers/user.go [hunk 1]
```diff
@@ -2,22 +2,23 @@ package controllers
...
 	"minitwit/src/models"
 	"net/http"
+
+	"github.com/gin-gonic/gin"
 )
 
 func userHandlers(router *gin.Engine) {
 
-	router.GET("/:user/follow", func(c *gin.Context) {
+	router.POST("/:user/follow", func(c *gin.Context) {
```
Comment (Thomsen96) [inline src/controllers/user.go:16]: This should be GET requests. according to the [Minitwit.py](https://github.com/DevelOpsITU/MiniTwit/blob/48dcde50e0360203d7e278c7fc85f180c54cb384/minitwit.py#L140)

Where absent request method means GET.

File: src/controllers/user.go [hunk 2]
```diff
@@ -2,22 +2,23 @@ package controllers
...
 		username := c.Param("user")
 		handleFollowUser(c.Writer, c.Request, c, username)
 	})
 
-	router.GET("/:user/unfollow", func(c *gin.Context) {
+	router.POST("/:user/unfollow", func(c *gin.Context) {
```
Comment (Thomsen96) [inline src/controllers/user.go:21]: Same as above

**Review by Thomsen96 — APPROVED** [review]
LGTM

File: src/controllers/user.go (hunk 2 above)
Comment (Lucki2g) [reply src/controllers/user.go]: Was sure I changed this... must have forgotten to commit before creating this pull request

**Pull request merged**